src/composites/FeaturesContainer/Overview/OverviewCollector.ts

```diff
@@ -0,0 +1,107 @@
+import { OverviewData } from './index.ts';
+
+//Reads through dataset information and targets specific data
+// from it to display information more easily in surrounding 
+// overview functions.
+export function OverviewDataCollector(fileData):OverviewData {
```

Comment:
can we add a type for fileData (`schema.base.Dataset`)?

Reply:
also refer to my last review regarding name of this function

Reply:
Addressed from previous comments